File: challenge/top_interview_questions/easy_collection/array/python/remove_duplicates_from_sorted_array.py
class Solution:
	def removeDuplicates(self, nums: List[int]) -> int:
		counter = 0
		for i in range(1, len(nums)):
			if nums[counter] != nums[i]:
			nums[counter + 1], nums[i] = nums[i], nums[counter + 1]
			counter += 1
		return counter + 1 if nums else 0

File: challenge/top_interview_questions/easy_collection/array/cpp/remove_duplicates_from_sorted_array.cpp
class Solution {
	public:
		int removeDuplicates(vector<int>& nums) {
			int n = nums.size();
			int index = 0;

			if (n == 0)
				return 0;
			for (int i = 1; i < n; i++)
				if (nums[i] != nums[index])
					nums[++index] = nums[i];
			return index + 1;
		}
};
